### ISSUE TITLE
What's the best practice of LTI Context to Drupal group mapping and Memberships Groups?

### ISSUE THREAD

**Issue by fishfree** [PoeticMediaLab/LacunaStories#241]
I mean, after enabling the LTI features, how to configure on the pages:  admin/config/lti-tool-provider/group-mapping  and   admin/config/lti-tool-provider/memberships-groups

**Comment by mwidner** [comment]
Generally, the groups correspond to classrooms. So, if Moodle is sending classroom information via the LTI protocol, those should be the groups to map in Lacuna.

**Comment by fishfree** [comment]
@mwidner May I think "classroom" as "course" in Moodle? For, there is no concept of "classroom" in Moodle.

**Comment by mwidner** [comment]
Yes, that's what I meant. Sorry for the confusion. The groups in Lacuna map to a course.

**Comment by fishfree** [comment]
@mwidner Thank you! Execuse me, could you please paste your screenshots of admin/config/lti-tool-provider/group-mapping and admin/config/lti-tool-provider/memberships-groups?

**Comment by mwidner** [comment]
I've never set up a Moodle-Lacuna integration, so I'm afraid I don't have screenshots to share.

**Comment by fishfree** [comment]
@mwidner Canvas config screenshots also is OK. :-)

**Comment by mwidner** [comment]
Unfortunately, I no longer have access to Canvas for screenshots. @dabush can you provide some of the set up, by chance?
